src/main/java/it/gov/pagopa/pu/sil/connector/actualization/LegacyActualizationServiceImpl.java

```diff
@@ -20,7 +20,11 @@ public LegacyActualizationServiceImpl(LegacyActualizationClient legacyActualizat
 
   @Override
   public AmountUpdatesDTO actualization(String accessToken, String serviceUrl, Pagamento pagamento) {
-    PagamentoAggiornato actualization = legacyActualizationClient.actualization(accessToken, serviceUrl, pagamento);
-    return amountUpdatesMapper.pagamentoAggiornato2AmountUpdatesDTO(actualization);
+    try {
+      PagamentoAggiornato actualization = legacyActualizationClient.actualization(accessToken, serviceUrl, pagamento);
+      return amountUpdatesMapper.pagamentoAggiornato2AmountUpdatesDTO(actualization);
+    } catch (Exception e) {
+      return amountUpdatesMapper.mapToKoAmountUpdatesDTO();
```

Comment:
let's catch just non blocking exception: introduce a specific exception to differentiate between blocking and not blocking exception